readthedocs/projects/models.py

```diff
@@ -175,8 +175,8 @@ class Project(models.Model):
         default=LATEST,
         help_text=_('The version of your project that / redirects to'),
     )
-    # In default_branch, None means the backend should choose the
-    # appropriate branch. Eg 'master' for git
+    # In default_branch, ``None`` means the backend will use the default branch
+    # clonned for each backend.
```

Comment:
```suggestion
    # cloned for each backend.
```

Just a note -- you seem to misspell this one consistently :)

Reply:
I do certain things consistently wrong, too, and I wondered if somehow pre-commit could support private linting :joy:

Reply:
Yes. Every time that you see me writing is correctly, it's because I just fixed. I always write it incorrectly the first time 🙃 

> I wondered if somehow pre-commit could support private linting

I'm sure there is a spell checker we could easily add to pre-commit.